.github/workflows/development-client-e2e.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: macos-11
     strategy:
       matrix:
-        api-level: [ 29 ]
+        api-level: [ 31 ]
```

Comment:
Why do we have `[31]` in some places and `[ 31 ]` in others? 😄

Reply:
There is no particular reason, can update this to one of the formats. Which one do you prefer?